src/state/slices/portfolioSlice/portfolioSlice.ts

```diff
@@ -179,9 +182,21 @@ export const portfolioApi = createApi({
         try {
           const adapter = chainAdapters.get(chainId)
           if (!adapter) throw new Error(`no adapter for ${chainId} not available`)
-          const portfolioAccounts = { [pubkey]: await adapter.getAccount(pubkey) }
+          // "Fetch" the query leveraging the existing cached data
+          const portfolioAccounts = {
+            [pubkey]: await queryClient.fetchQuery({
+              ...accountManagement.getAccount(accountId),
+              staleTime: GET_ACCOUNT_STALE_TIME,
+              // Never garbage collect me, I'm a special snowflake
+              gcTime: Infinity,
+            }),
+          }
+
```

Comment:
Leverages the `getAccount` query instead of `adapter.getAccount()`.
This is one of the guts of this PR, meaning that this query will *not* be marked stale for `GET_ACCOUNT_STALE_TIME` (5mn currently) and not garbage-collected either without listeners. 

This ensures that next consumers (import accounts drawer in this case) will be able to leverage said cached data, and don't be making any further calls.